package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+# Warer Bridge Analysis
+r"""Warer Bridge analysis --- :mod:`MDAnalysis.analysis.hbonds.WaterBridgeAnalysis`
+===========================================================================
+
+:Author: Zhiyi Wu
+:Year: 2017
+:Copyright: GNU Public License v3
+
+
+Given a :class:`~MDAnalysis.core.universe.Universe` (simulation
+trajectory with 1 or more frames) measure all water bridges for each
+frame between selections 1 and 2.
```

Comment:
Briefly describe what a water bridge is. If you have academic citations, add them.

Reply:
Focus your description on water bridge analysis.

Make sure that users will immediately understand what problem you are solving here. Be explicit. If you need to show equations, do it! Academic references are great, too, e.g., where you found the definition of water bridges that you're using here.

Reply:
@orbeckst Sorry I cannot find any academic citations for water bridge. I have dug up papers as early as 1988 and they sort of just use the concept of water bridge without citing it.
https://www.ncbi.nlm.nih.gov/pubmed/2845953